packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -139,6 +139,13 @@ export interface DesignSystem {
     neutralFillStealthActiveDelta: number;
     neutralFillStealthSelectedDelta: number;
 
+    /**
+     * Configuration for the neutral-fill-toggle recipe.
+     */
+    neutralFillToggleRestContrast: number;

```

Comment:
I think we will expose those soon. Perhaps there will be another way we want to support increased contrast, but for instance, to coordinate with Mac increased contrast mode, we will want to support increasing the default 4.5 up to 7.

In this case I wasn't thinking of it being tied as directly to W3C, but perhaps the base recommendation is. In testing the design for these components we evaluated lighter or darker values. I was considering this the equivalent of a "rest" delta like the other recipes.

Long term I would like to see a better way to define and configure recipes, which this falls into. For instance, maybe I want my outline recipe to be contrast based instead of offset based. Perhaps we accept either a contrast or an offset and let the recipe resolve from there.

I'm fine hard-coding it like the other recipes until we have a chance to think about a wider strategy.

Reply:
Yea let's hard-code for now. When we figure the story out we can update holistically.